beta/src/pages/learn/index.md

```diff
@@ -4,716 +4,491 @@ title: Quick Start
 
 <Intro>
 
-Welcome to the React documentation! Here is an overview of what you can find on this site.
+Welcome to the React documentation! This page will give you an introduction to the 80% of React concepts that you will use on a daily basis.
 
 </Intro>
 
 <YouWillLearn>
 
-* [How to set up React](/learn/installation)
-* [How to think about building user interfaces with React](/learn/thinking-in-react)
-* [How to get something rendering on the screen](/learn/describing-the-ui)
-* [How to make your user interface respond to interactions](/learn/adding-interactivity)
-* [How to keep your logic maintainable as your app grows](/learn/managing-state)
-* [How to step outside React when necessary](/learn/escape-hatches)
+- How to create and nest components
+- How to add markup and styles
+- How to display data
+- How to render conditions and lists
+- How to respond to events and update the screen
+- How to share data between components
 
 </YouWillLearn>
 
-## Introduction {/*introduction*/}
+## Creating and nesting components {/*components*/}
 
-This is a tiny React app. To get your first taste of React, **edit the code below** and make it display your name:
+React apps are made out of components. A component is a piece of the UI (user interface) that has its own logic and appearance. A component can be as small as a button, or as large as an entire page.
 
-<Sandpack>
+React components are JavaScript functions that return markup:
```

Comment:
**problem**: "Markup" is not a term commonly used in tech curricula. Also, if you google "markup programming", you get an umbrella-term explanation. 

**possible solutions**:
- explain what markup is,
- link to a definition,
- find a synonym

Reply:
One thing I've been trying to avoid here is having too many links so that people don't get distracted with rabbit holes. "Markup" seems commonly known as in we assume the knowledge of HTML, which is a "markup language", and HTML tags are commonly referred to as "markup".

It _is_ a bit intentionally vague since the "markup" we refer to is actually JSX and not HTML. (But we haven't introduced JSX yet.) This is why we don't say "functions that return HTML".

Ultimately I don't think this one is an issue because even if you don't know what markup is, the code example serves to define it. ("Aha, `<button>Click me</button>` is markup"). But if you have alternative wordings (I prefer not to introduce new links here) I'm open to it.